project/src/main/java/com/google/autograder/servlets/auth/ExchangeCodeServlet.java

```diff
@@ -20,13 +20,13 @@
 import com.google.appengine.api.datastore.Entity;
 import com.google.autograder.servlets.helpers.API;
 
-@WebServlet("/exchangeAuthCode")
+// @WebServlet("/exchangeAuthCode")
 public final class ExchangeCodeServlet extends HttpServlet {
 
     private static final String CLIENT_ID = "361755208772-l0oo78304ot5ded6rb0tgbhjhrqgmc53.apps.googleusercontent.com";
     private static final String CLIENT_SECRET = "jbzCa4-vkwu394TEk9PEnqNj";
-
-    private static String HOST_URL = "https://8080-778d1d95-8447-4f8a-990b-b90da194d107.us-east1.cloudshell.dev";
+    
+    private static String BASE_URL = "https://8080-778d1d95-8447-4f8a-990b-b90da194d107.us-east1.cloudshell.dev"; // "https://step25-2020.uc.r.appspot.com"; 
```

Comment:
The base URL shouldn't be needed since you can access it from the HttpServletRequest

Reply:
Oh, you're very right here. This will also make it much easier to deploy as well because it'll be a lot simpler to authenticate.

Reply:
+1 This will be handled in a later PR.